README.md

```diff
@@ -219,22 +294,28 @@ And the `(A : Type;) -> (x : A) -> A` term would be represented as:
     "body": {
       "ctor": "Var",
       "indx": 1
-    },
-    "eras": false
-  },
-  "eras": true
+    }
+  }
 }
 ```
 
+or
+
+```
+(All True "A" Typ (All False "x" (Var 0) (Var 1)))
+```
+
 In reference implementations, parsing is done through a combination of small
 backtracking parsers that receive the code to be parsed and return either a pair
 with the leftover code and parsed value, or throw if the parser failed. The term
 parser is divided in two phases: first, a base term is parsed, including the
 variants `Var`, `Typ`, `All`, `Lam`, `Slf`, `Ins` and `Eli`. Afterwards, the
 term is extended by postfix parsers, including the variants `App` and `Ann`.
 Parsing involves subtle details that could differ among implementations. To
-prevent that, the reference implementations included should be considered the
-specification.
+prevent confusion, the reference implementations in this repository are provided
+to disambiguate and specificy the correct semantic behavior. These reference
+parsers are *not* designed to be performant, and practical Formality parsers are
+expected to differ significantly from them in implementation detail.
```

Comment:
I'd remove the performance comment here, for now (they're turning out to be really fast btw). I like the rephrasing

Reply:
Okay, I'll remove the performance comment. The idea is that the reference parsers are designed to specify semantics, but that implementations are free to differ in structure/design as long as they do the same thing